### ISSUE TITLE
pyscf_lib_placeholder.so is written into plist but isn't installed

### ISSUE THREAD

**Issue by yurivict** [pyscf/pyscf#973]
```setup.py install --record {plist}``` writes ```lib/python3.8/site-packages/pyscf_lib_placeholder.so``` into plist but doesn't install it.

FreeBSD ports framework complains:
```
====> Checking for pkg-plist issues (check-plist)
===> Parsing plist
===> Checking for items in STAGEDIR missing from pkg-plist
===> Checking for items in pkg-plist which are not in STAGEDIR
Error: Missing: %%PYTHON_SITELIBDIR%%/pyscf_lib_placeholder.so
===> Error: Plist issues found.
```

**Comment by verena-neufeld** [comment]
Hi @yurivict , thanks for having raised this issue. Since this was a few releases ago, has this been resolved by now?

**Comment by sunqm** [comment]
Fix in PR #1900
